frontend/src/components/Project/Settings/ProjectGeneralSettingPanel.vue

```diff
@@ -20,13 +20,88 @@
         />
       </div>
     </div>
+
+    <!-- Project Labels Section -->
+    <div>
```

Comment:
Can we reuse the `Labels` component in the `components/DatabaseDetail/Settings/components/Labels.vue`?